src/motor_control.c

```diff
@@ -51,12 +57,14 @@ static inline void reset_tone(MotorControl *mc) {
     mc->tone_high = 0;
 }
 
-void motor_control_apply(MotorControl *mc, float abs_erpm, RunState state, float time) {
-    if (mc->parking_brake_mode == PARKING_BRAKE_ALWAYS ||
-        (mc->parking_brake_mode == PARKING_BRAKE_IDLE && state != STATE_RUNNING && abs_erpm < 50)) {
-        mc->parking_brake_active = true;
-    } else if (mc->parking_brake_mode == PARKING_BRAKE_NEVER || state == STATE_RUNNING) {
-        mc->parking_brake_active = false;
+void motor_control_apply(MotorControl *mc, const MotorData *md, RunState state, float time) {
```

Comment:
`const MotorData *motor` is used elsewhere, though I personally think `*md` is best:

https://github.com/lukash/refloat/blob/fe9d7b4a2c80432d4913a2c58e58eaf2637c78b9/src/torque_tilt.c#L34

https://github.com/lukash/refloat/blob/fe9d7b4a2c80432d4913a2c58e58eaf2637c78b9/src/atr.c#L53

https://github.com/lukash/refloat/blob/fe9d7b4a2c80432d4913a2c58e58eaf2637c78b9/src/atr.c#L203

https://github.com/lukash/refloat/blob/fe9d7b4a2c80432d4913a2c58e58eaf2637c78b9/src/torque_tilt.h#L35

----

(`*m` used in other places, so maybe it isn't important, yet?)

https://github.com/lukash/refloat/blob/fe9d7b4a2c80432d4913a2c58e58eaf2637c78b9/src/atr.h#L47

https://github.com/lukash/refloat/blob/fe9d7b4a2c80432d4913a2c58e58eaf2637c78b9/src/motor_data.c#L39

Reply:
I tried to be consistent within the line of code I changed, so *md makes sense. But idk why you would even bother commenting on stuff like this. I could care less. Let Lukas name it however he wants.

Reply:
> idk why you would even bother commenting on stuff like this

Because that is how contributing to open-source software works, and how the code gets better over time.

The world reviews requests for progress, and iterates forward together.

>  I could care less

That's your problem; don't make it mine.

Reply:
It would be nice to have those consistent, but I'm not gonna spend time going over them to try to unify everything. Sometimes it's a balance of a short name that'll keep the lines from wrapping vs. the name being descriptive enough.

And I'm not gonna bug the PR author with such a small thing if the name is not outright bad.

If you want, @JJJ, you can post a PR unifying all the names of these arguments, I'll merge it :grin: